component-catalog/src/Examples/UiIcon.elm

```diff
@@ -89,6 +89,9 @@ all =
         , ( "gift", UiIcon.gift, [] )
         , ( "openInNewTab", UiIcon.openInNewTab, [] )
         , ( "sync", UiIcon.sync, [] )
+        , ( "delete", UiIcon.delete, [])
+        , ( "addSticker", UiIcon.addSticker, [])
+        , ( "chooseColor", UiIcon.chooseColor, [])
```

Comment:
Maybe "circle" instead of "chooseColor" if we wanted to use this more generically later? Though I can't think of what else we'd use a circle icon for vs. drawing it as a shape (e.g. bullet points, background to place a different icon on top of)

Reply:
Done!